test/user.test.js

```diff
@@ -787,8 +794,12 @@ describe('User', function() {
     });
   });
 
-  function assertGoodToken(accessToken) {
-    assert(accessToken.userId);
+  function assertGoodToken(accessToken, user) {
+    if (accessToken instanceof AccessToken) {
+      console.log('its instanceof');
+      accessToken = accessToken.toJSON();
+    }
+    expect(accessToken).to.have.property('userId', user.pk);
     assert(accessToken.id);
     assert.equal(accessToken.id.length, 64);
   }
```

Comment:
I refactored the assertGoodToken() helper to test the actual value of the accessToken.userId property
the method signature now take a user object.
we flatten toJSON the accessToken object if is an instance of AccessToken
this enhanced helper will also serve when testing multiple-user-with-custom-principalType feature
ps. i forgot to remove one console.log(), but reverted in the next commit